### PR TITLE
hello.py: add insecure registry flag and timestamp

### LINKED ISSUE #3: [feat]insecure-registry
I don't find any args can use the insecure-registry.
When testing in a private registry like harbor, we always need to skip verify, so maybe we can add the arg  --insecure(bool, default false) to use the `nerdctl pull --insecure-registry xxxx` as the pull_cmd when insecure is true. As follows(not done).
image

### PR DESCRIPTION
**Pull request by sctb512**
And add output format flag. Currently, there are two available options: json and csv.

This PR tries to fix #3.

Signed-off-by: Bin Tang <tangbin.bin@bytedance.com>

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -870,22 +892,41 @@ def main():
         registry2=registry2,
         snapshotter=snapshotter,
         cleanup=cleanup,
+        insecure_registry=insecure_registry,
     )
 
+    if output_format == "csv":
+        csv_headers = (
+            "timestamp,repo,bench,pull_time(s),create_time(s),run_time(s),elapsed(s)"
```
Comment (changweige) [inline hello.py:900]: pull_times(s) looks strange, is `time` countable?

File: hello.py [hunk 2]
```diff
@@ -870,22 +892,41 @@ def main():
...
+                "create_time": cleate_time,
+                "run_time": run_time,
+                "elapsed": elapsed_time,
+            }
+            line = json.dumps(row)
+        elif output_format == "csv":
+            line = f"{timetamp},{bench.repo},{bench.name},{pull_time},{cleate_time},{run_time},{elapsed_time}"
+
+        print(line)
+        f.write(line + "\n")
```
Comment (changweige) [inline hello.py:929]: Try `writelines`

File: hello.py (hunk 1 above)
Comment (sctb512) [reply hello.py]: Yes, I have replaced it with `elapsed`.

File: hello.py (hunk 2 above)
Comment (sctb512) [reply hello.py]: Fixed. I did not make a design to write once. Because all datas will be lost when one task fails.

**Review by changweige — APPROVED** [review]
LGTM

**Pull request merged**